fjord/base/management/commands/extract.py

```diff
@@ -15,48 +14,29 @@
 
 
 DEFAULT_DOMAIN = 'all'
-TEXT_DOMAIN = getattr(settings, 'TEXT_DOMAIN', 'messages')
-
-
-# JINJA_CONFIG can be a callable or a dict.
-if hasattr(settings.JINJA_CONFIG, '__call__'):
-    JINJA_CONFIG = settings.JINJA_CONFIG()
-else:
-    JINJA_CONFIG = settings.JINJA_CONFIG
-
-
-# By default, all the domains you speficy will be merged into one big
-# messages.po file.  If you want to separate a domain from the main .po file,
-# specify it in this list.  Make sure to include TEXT_DOMAIN in this list, even
-# if you have other .po files you're generating
-try:
-    standalone_domains = settings.STANDALONE_DOMAINS
-except AttributeError:
-    standalone_domains = [TEXT_DOMAIN]
-
-
-TOWER_KEYWORDS = dict(DEFAULT_KEYWORDS)
-
-
-if hasattr(settings, 'TOWER_KEYWORDS'):
-    TOWER_KEYWORDS.update(settings.TOWER_KEYWORDS)
+TEXT_DOMAIN = 'django'
+KEYWORDS = dict(DEFAULT_KEYWORDS)
+KEYWORDS.update({'_lazy': None})
```

Comment:
It's possible I had more stuff in there, but then nixed it later. I'm not really sure. I don't see anything in my notes. I'll change it since the `KEYWORDS['_lazy'] = None` is less curious looking.